SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
-SKILL_20151224_006011
-SKILL_20151224_006012
-SKILL_20151224_006013
-SKILL_20151224_006014
-SKILL_20151224_006015
-SKILL_20151224_006016
-SKILL_20151224_006017
-SKILL_20151224_006018
-SKILL_20151224_006019
-SKILL_20151224_006020
-SKILL_20151224_006021
-SKILL_20151224_006022
-SKILL_20151224_006023
-SKILL_20151224_006024
-SKILL_20151224_006025
-SKILL_20151224_006026
-SKILL_20151224_006027
-SKILL_20151224_006028
-SKILL_20151224_006029
-SKILL_20151224_006030
-SKILL_20151224_006031
-SKILL_20151224_006032
+SKILL_20151224_005737	Mining
+SKILL_20151224_005738	Increase block count: #{CaptionRatio}#{nl}Attack reduction: #{CaptionTime}#%{nl}Critical hit rate reduction: #{CaptionRatio2}#{nl}Duration: 30 seconds
+SKILL_20151224_005739	Langort
+SKILL_20151224_005740	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Block and strike simultaneously by using both a sword and a shield.
+SKILL_20151224_005741	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Attack with descending vertical slash. Additional damage dealt onto defense reduced medium type and large type enemies from Scull Swing. Depending on the amount of harmful effect, additional damage is applied.
+SKILL_20151224_005742	Increases attack equal to [Skill Level+1]x[Number of Weapon Stars]{nl}Attack Count: #{CaptionRatio2}#Duration: 1 hour
+SKILL_20151224_005743	Penalty Reduction
+SKILL_20151224_005744	Reduce the penalty from combat incapability
+SKILL_20151224_005745	Equipment durability penalty reduction: #{CaptionRatio}#%{nl}Silver consumption penalty reduction: #{CaptionRatio2}#%{nl}Gem consumption penalty reduction: #{CaptionTime}#%
+SKILL_20151224_005746	Plant a flag that raises the combat abilities of the pirate crew. A combo is activated if multiple enemies is defeated around the jolly roger. Also at certain rate, attacking can pillage items from enemies and the rate is increased as Jolly Roger level is raised. (Combo is only activated if the party leader is Corsair.)
+SKILL_20151224_005747	Pistol Shot
+SKILL_20151224_005748	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire continuous shots with pistol to damage multiple enemies ahead.
+SKILL_20151224_005749	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Slash enemies with powerful swing to damage the enemy.
+SKILL_20151224_005750	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Spin the weapon to continuously damage the enemy in front of you.
+SKILL_20151224_005751	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Damage the enemy by slashing multiple times from side to side.
+SKILL_20151224_005752	Attack enemies in the area by exploding the gunpowder in hand. This technique will also damage the user and might cause combat incapability if the HP is too low.
+SKILL_20151224_005753	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: %{CaptionRatio}#%{nl}HP consumption: #{CaptionRatio2}#
+SKILL_20151224_005754	Creates clones of yourself. Clones will act along with the main body and skills can be used along with the clones. HP of the clones is determined by the user's dexterity.
```

Comment:
Create clones of yourself. The clones will act along as a main body and skills can be used along with your clones. The HP of your clones is determined by your DEX.